QueryBuilder.Test/QueryBuilder.Typed/Select.UnitTests.cs

```diff
@@ -1,12 +1,13 @@
 ﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace QueryBuilder.UnitTests
+namespace QueryBuilder.UnitTests.QueryBuilder.Typed
```

Comment:
Not sure, I must have gotten a bit lost in what I was doing. I'll correct this.

Reply:
Turns out this is actually because I was aligning the namespace to the folder path as we'd done in other areas. Is there a different folder structure/naming you'd suggest here?